sssd_test_framework/hosts/ldap.py

```diff
@@ -23,10 +23,17 @@ class LDAPHost(BaseLDAPDomainHost):
         Full backup and restore is supported.
     """
 
+    def _start_and_wait(self) -> None:
+        # systemctl finishes before ldap is fully started, wait for it to start listening on ldap port
+        self.ssh.run(f"systemctl start {self._ldap_service_name}")
+        self.ssh.run("timeout 60s bash -c 'until ss -ltp 2> /dev/null | grep :ldap &> /dev/null; do :; done'")
```

Comment:
It might be better to pipe it with && or put into a single script differently to save some cycles. But I don't insist. E.g.:

f"""
set -e
systemctl start ...
timeout ...
"""

Reply:
To me adding multiline-inline scripts looks like when we were creating and executing bash files.
I was thinking if we should not write some  helper function like "wait for port/service open" instead.

Reply:
A wait for port/service helper function could be useful in other places (like Keycloak) where it takes an unpredictable and noticeable amount of time for a service to start.   I don't want to hold up this PR just for that though.

Reply:
Well, in reality, you are executing bash script. So there is no problem with that.

Reply:
The wait is not needed after all. Dropped the code.